frontend/dockerfile/dockerfile_test.go

```diff
@@ -175,8 +175,6 @@ var allTests = integration.TestFuncs(
 	testDuplicatePlatformProvenance,
 	testDockerIgnoreMissingProvenance,
 	testSBOMScannerArgs,
-	testLintStageName,
-	testLintNoEmptyContinuations,
 	testSelfConsistentCommandCasing,
```

Comment:
I guess these should be defined in `lint_test.go` now as this is where the definition has been moved to.

Reply:
Looks like this was not updated

Reply:
Slippery. Gottem.

Reply:
You forgot to delete to old ones so they appear twice now.